<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BalanceForm</class>
 <widget class="QWidget" name="BalanceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>转向</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="CMBoxClockwise">
               <item>
                <property name="text">
                 <string>顺时针</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>逆时针</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>键相位置°</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="DSBoxKeyPos">
               <property name="minimum">
                <double>-360.000000000000000</double>
               </property>
               <property name="maximum">
                <double>360.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>X向位置°</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="DSBoxXPos">
               <property name="minimum">
                <double>-360.000000000000000</double>
               </property>
               <property name="maximum">
                <double>360.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Y向位置°</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="DSBoxYPos">
               <property name="minimum">
                <double>-360.000000000000000</double>
               </property>
               <property name="maximum">
                <double>360.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>X向幅值μm (Ax)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="DSBoxXamp">
               <property name="maximum">
                <double>1000.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>X向相位°  (Ax)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="DSBoxXpha">
               <property name="minimum">
                <double>-360.000000000000000</double>
               </property>
               <property name="maximum">
                <double>360.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>X向滞后角  (Ax)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="DSBoxXlag">
               <property name="minimum">
                <double>-360.000000000000000</double>
               </property>
               <property name="maximum">
                <double>360.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Y向幅值μm (Ay)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="DSBoxYamp">
               <property name="maximum">
                <double>1000.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Y向相位°  (Ay)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="DSBoxYpha">
               <property name="minimum">
                <double>-360.000000000000000</double>
               </property>
               <property name="maximum">
                <double>360.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Y向滞后角  (Ay)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="DSBoxYlag">
               <property name="minimum">
                <double>-360.000000000000000</double>
               </property>
               <property name="maximum">
                <double>360.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>按X向加重位置
(键相逆转向)°(Qx)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="TextXweightpos">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
               <property name="dragEnabled">
                <bool>true</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>按Y向加重位置
(键相逆转向)°(Qy)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="TextYweightpos">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="dragEnabled">
                <bool>true</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>转子质量kg</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="DSBoxMass">
               <property name="maximum">
                <double>100000.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>转速rpm</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="DSBoxRPM">
               <property name="maximum">
                <double>100000.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>加重半径mm</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="DSBoxRadius">
               <property name="maximum">
                <double>15000.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>试加重质量g</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="DSBoxWeight">
               <property name="maximum">
                <double>100000.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="9,4">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>加重1g产生激振力kgf    </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="TextUnitKgf">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="dragEnabled">
                <bool>true</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="9,4">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>试加重产生总的激振力kgf</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="TextAllKgf">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="dragEnabled">
                <bool>true</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="9,4">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>试加重激振力与转子重力比%</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="TextPercent">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="dragEnabled">
                <bool>true</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4" stretch="15,2">
           <item>
            <layout class="QVBoxLayout" name="LayoutTab1Plot"/>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout" rowstretch="1,0">
             <item row="0" column="4">
              <widget class="QCheckBox" name="CheckDispAy">
               <property name="text">
                <string>显示Ay</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="8">
              <widget class="QCheckBox" name="CheckDispPyLabel">
               <property name="text">
                <string>显示Py标签</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="7">
              <widget class="QCheckBox" name="CheckDispPy">
               <property name="text">
                <string>显示Py</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QCheckBox" name="CheckDispAyLabel">
               <property name="text">
                <string>显示Ay标签</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="CheckDispAx">
               <property name="text">
                <string>显示Ax</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QCheckBox" name="CheckDispPx">
               <property name="text">
                <string>显示Px</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QCheckBox" name="CheckDispPxLabel">
               <property name="text">
                <string>显示Px标签</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="CheckDispAxLabel">
               <property name="text">
                <string>显示Ax标签</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="8">
              <widget class="QPushButton" name="BTNHelp">
               <property name="text">
                <string>帮助</string>
               </property>
              </widget>
             </item>
             <item row="1" column="7">
              <widget class="QCheckBox" name="CheckDispAll">
               <property name="text">
                <string>显示所有</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QCheckBox" name="CheckDispQx">
               <property name="text">
                <string>显示Qx</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QCheckBox" name="CheckDispQy">
               <property name="text">
                <string>显示Qy</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="CheckDispAxAngle">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Ax角注</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="CheckDispPxAngle">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Px角注</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QCheckBox" name="CheckDispAyAngle">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Ay角注</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QCheckBox" name="CheckDispPyAngle">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Py角注</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
